ftl/common/builder.py

```diff
@@ -113,41 +110,24 @@ def __init__(self, ctx, namespace, args, cache_version_str,
     def Build(self):
         return
 
-    def AppendLayersIntoImage(self, lyr_imgs):
-        for i, lyr_img in enumerate(lyr_imgs):
-            if i == 0:
-                try:
-                    result_image = lyr_img.GetImage()
-                except AttributeError:
-                    result_image = lyr_img
-                continue
-            img = lyr_img.GetImage()
-            diff_ids = img.diff_ids()
-            for diff_id in diff_ids:
-                lyr = img.blob(img._diff_id_to_digest(diff_id))
-                overrides = ftl_util.CfgDctToOverrides(
-                    json.loads(img.config_file()))
-                result_image = append.Layer(
-                    result_image, lyr, diff_id=diff_id, overrides=overrides)
-        return result_image
-
     def StoreImage(self, result_image):
-        if self._args.output_path:
-            with ftl_util.Timing("saving_tarball_image"):
-                with tarfile.open(
-                        name=self._args.output_path, mode='w') as tar:
-                    save.tarball(self._target_image, result_image, tar)
-                logging.info("{0} tarball located at {1}".format(
-                    str(self._target_image), self._args.output_path))
-            return
-        if self._args.upload:
-            with ftl_util.Timing("pushing_image_to_docker_registry"):
-                with docker_session.Push(
-                        self._target_image,
-                        self._target_creds,
-                        self._transport,
-                        threads=_THREADS,
-                        mount=[self._base_name]) as session:
-                    logging.info('Pushing final image...')
-                    session.upload(result_image)
+        with ftl_util.Timing('Uploading final image'):
```

Comment:
Did anything actually change here? This looks mostly cosmetic.

Reply:
No, I got a little carried away with making the `Timing` strings consistent while waiting for builds to finish. Should have done that in a separate commit/PR